stories/dynamic-page/dynamic-page.stories.js

```diff
@@ -0,0 +1,950 @@
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+            </div>
+        </div>
+        <div class="fd-bar fd-bar--floating-footer">
+            <div class="fd-bar__right">
+                <div class="fd-bar__element">
+                    <button aria-label="button" class="fd-button fd-button--emphasized fd-button--compact">Save</button>
+                </div>
+                <div class="fd-bar__element">
+                    <button aria-label="button" class="fd-button fd-button--transparent fd-button--compact">Cancel</button>
+                </div>
+            </div>
+        </div>
+    </div>
+    <br /><br />
+`;
+
+desktop.storyName = 'Desktop mode';
+desktop.parameters = {
+    docs: {
+        iframeHeight: 150,
+        storyDescription: `The dynamic page displays on desktop screens with the L, XL, and XXL sizes.
+        **Tabs** can be optionally added as an extension of the header area with the tab content being a part of the page content. 
+        Please note the difference in the way tabs are used. Tabs must use \`fd-dynamic-page__tabs\` along with the size modifiers(--m, --l, --xl or --s) and
+        \`--add-shadow\` at the \`<ul>\` level, and to ensure the correct box-shadow styling, should also add \`--overflow\` modifier at the parent level.
+        To optimize dynamic page for desktop, add these modifier classes to the following elements:
```

Comment:
The dynamic page displays on desktop screens with the L, XL, and XXL sizes. To optimize dynamic page for desktop, add these modifier classes to the following elements:

| Element | Modifier class |
| ----------------: | :------------ |
| Title Area | `fd-dynamic-page__title-area--xl` |
| Header | `fd-dynamic-page__header--xl` |
| Tabs | `fd-dynamic-page__tabs--xl` |
| Content | `fd-dynamic-page__content--xl` |

Note: The XL size is presented in this table, however the L and XXL size works as well.

It is also possible to add **Tabs** to the header area, which helps divide the page content. To display tabs, add the `fd-dynamic-page__tabs` class along with any of these size modifiers:
- `--s`
- `--m`
- `--l`
- `--xl`

Also add the modifier classes `--add-shadow` to the unordered list, and `--overflow` at the parent level for the box-shadow styling.

Reply:
modified accordingly.